### PR TITLE
No markdown inside html

### PR DESCRIPTION
**Pull request by nikku**

### CODE REVIEW

File: packages/form-js-viewer/src/render/components/Sanitizer.js [hunk 1]
```diff
@@ -119,6 +121,11 @@ function sanitizeNode(node) {
 
   }
 
+  // force noopener on target="_blank" links
```
Comment (philippfromme) [inline packages/form-js-viewer/src/render/components/Sanitizer.js:124]: Good stuff!

**Review by philippfromme — APPROVED** [review]
Great improvement! 🏅

**Comment by philippfromme** [comment]
I even managed to approve in time! 🕐

**Comment by nikku** [comment]
Great achievement.

**Pull request merged**